run_local.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
+
+if [ -z ${SECRET_KEY} ]; then
+    echo "SECRET_KEY is unset."
+    exit 1
+fi
+
 docker compose down --remove-orphans
 docker compose up -d --build
 docker compose wait migrations
 docker compose logs migrations
 docker compose rm --force
 docker compose logs api
+python3 bin/add_users.py
```

Comment:
Not sure why. This seems to be causing an error for me locally when I do ./run_local.sh 
Traceback (most recent call last):
  File "/Users/javidali/MOJ/laa-civil-case-api/bin/add_users.py", line 7, in <module>
    from app.db import get_session
  File "/Users/javidali/MOJ/laa-civil-case-api/app/__init__.py", line 1, in <module>
    from app.main import create_app
  File "/Users/javidali/MOJ/laa-civil-case-api/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'

Reply:
Hiya Jav, have you set up a virtual machine?